Lab2/ComputerStore.BusinessLogicLayer/Validation/Validator.cs

```diff
@@ -1,81 +1,10 @@
-﻿using ComputerStore.BusinessLogicLayer.Exception;
-using ComputerStore.DataAccessLayer.Models;
-using ComputerStore.DataAccessLayer.Repo;
-using System;
-using System.Collections.Generic;
+﻿using ComputerStore.DataAccessLayer.Models;
 
 namespace ComputerStore.BusinessLogicLayer.Validation
 {
     public abstract class Validator<T>
```

Comment:
It would be better to have an interface IValidator<> instead abstract class.
Also, you need to fill constructors in Managers - they should request IValidator<Product>  instead ProductValidator.